src/components-examples/material/button-toggle/button-toggle-forms/button-toggle-forms-example.ts

```diff
@@ -0,0 +1,13 @@
+import {Component} from '@angular/core';
+import {FormControl} from '@angular/forms';
+
+/**
+ * @title Button-toggles with forms
```

Comment:
Should the title say that it's with reactive forms specifically? Same goes for the file name.

Reply:
Now I think maybe there's sense to add example with template-driven forms also.
What do you think?

Reply:
Sure, if you're up for it.

Reply:
I've added an example with template-driven forms